### PR TITLE
Ygutierrez sem 

### PR DESCRIPTION
**Pull request by yoangutierrez**
Creating folders for redirecting three URL Ontologies.

### CODE REVIEW

**Comment by yoangutierrez** [comment]
Hi, could I know why the checks have failed? As far I know the branch does not has conflicts. Thanks

**Comment by dgarijo** [comment]
It's fine, that is due to some other issues caused by URIs with rawgit.

File: nlp/semanticpackage/.htaccess [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://gplsi.dlsi.ua.es/gplsi13/sites/default/files/resources/semanticpackage.owl [R=302,L]
+RewriteRule ^_1.1$ https://gplsi.dlsi.ua.es/gplsi13/sites/default/files/resources/semanticpackage_1.1.owl [R=302,L]
+RewriteRule ^_bucket_BarcelonaOpen2015$ https://gplsi.dlsi.ua.es/gplsi13/sites/default/files/resources/semanticpackage_bucket_BarcelonaOpen2015.owl [R=302,L]
```
Comment (davidlehn) [inline nlp/semanticpackage/.htaccess]: I don't think these do what you mean to do.  They will match paths like `/nlp/semanticpackage/_bucket_BarcelonaOpen2015`.  Yet in README you list `/nlp/semantic_bucket_BarcelonaOpen2015`.

File: nlp/semanticpackage/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+===
+
+Homepage:
+* https://gplsi.dlsi.ua.es
+
+Vocabularies:
+* https://w3id.org/nlp/semanticpackage (@prefix sem: https://w3id.org/nlp/semanticpackage#)
+
+Instances: 
+* https://w3id.org/nlp/semanticpackage_bucket_BarcelonaOpen2015 (@prefix sem: https://w3id.org/nlp/semanticpackage_bucket_BarcelonaOpen2015#)
```
Comment (davidlehn) [inline nlp/semanticpackage/README.md]: Your rules don't setup these URLs and it causes test failures.

**Comment by davidlehn** [comment]
In this case it's not just the rawgit issue for other redirs.  The rules and README URLs in this PR fail.

@yoangutierrez: Please do another PR with fixes for this issue, thanks.

File: nlp/semanticpackage/.htaccess (hunk 1 above)
Comment (dgarijo) [inline nlp/semanticpackage/.htaccess]: My bad, I didn't realize about this

**Comment by dgarijo** [comment]
@davidlehn 
Shall we revert the PR until then?

**Comment by davidlehn** [comment]
Easier to just wait a while for a fix first.

**Pull request merged**